src/components/v2/V2Project/V2FundingCycleSection/ReservedTokensSplitsCard.tsx

```diff
@@ -138,21 +143,41 @@ export default function ReservedTokensSplitsCard({
         )}
 
         <div>
-          <TooltipLabel
-            label={
-              <h4 style={{ display: 'inline-block' }}>
-                <Trans>Reserved {tokensText}</Trans> (
-                {formatReservedRate(reservedRate)}%)
-              </h4>
-            }
-            tip={
-              <Trans>
-                A project can reserve a percentage of tokens minted from every
-                payment it receives. Reserved tokens can be distributed
-                according to the allocation below at any time.
-              </Trans>
-            }
-          />
+          <div
+            style={{
+              display: 'flex',
+              justifyContent: 'space-between',
+              gap: 10,
+              flexWrap: 'wrap',
+            }}
+          >
+            <TooltipLabel
+              label={
+                <h4 style={{ display: 'inline-block' }}>
+                  <Trans>Reserved {tokensText}</Trans> (
+                  {formatReservedRate(reservedRate)}%)
+                </h4>
+              }
+              tip={
+                <Trans>
+                  A project can reserve a percentage of tokens minted from every
+                  payment it receives. Reserved tokens can be distributed
+                  according to the allocation below at any time.
+                </Trans>
+              }
+            />
+            {reservedRate?.gt(0) ? (
```

Comment:
**issue** we should only display this button if the connected user is the project owner.

Reply:
can probs re-use this https://github.com/jbx-protocol/juice-interface/pull/1256